src/types/marketplace.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {PluginManifest} from './plugins';
+import {AppManifest} from './apps';
+
+export type MarketplaceLabel = {
+    name: string;
+    description?: string;
+    url?: string;
+    color?: string;
+}
+
+export enum HostingType {
+    OnPrem = 'on-prem',
+    Cloud = 'cloud',
+}
+
+export enum AuthorType {
+    Mattermost = 'mattermost',
+    Partner = 'partner',
+    Community = 'community',
+}
+
+export enum ReleaseStage {
+    Production = 'production',
```

Comment:
TODO: add a documentation ticket to detail what do these mean in the apps context?

Reply:
@catalintomai These fields will be used in the new Marketplace design. What kind of documentation are you referring to?